lib/components/artist/articles/article.js

```diff
@@ -3,25 +3,32 @@
 
 import Relay from 'react-relay';
 import React from 'react-native';
-const { ScrollView, StyleSheet, View, Text } = React;
+const { ScrollView, StyleSheet, View, Text, TouchableWithoutFeedback } = React;
 
 import colors from '../../../../data/colors';
 import SerifText from '../../text/serif';
 import ImageView from '../../opaque_image_view';
+import SwitchBoard from '../../../modules/switch_board';
 
 class Article extends React.Component {
+  handleTap() {
+    SwitchBoard.presentNavigationViewController(this, `${this.props.article.href}`);
```

Comment:
Btw, the href is already a string, right? So there shouldn’t be a need to interpolate it into another (backticks) string.